### ISSUE TITLE
Error building exampleSite - index of untyped nil

### ISSUE THREAD

**Issue by razvanf1** [LucasVadilho/heyo-hugo-theme#1]
Hello!
I just wanted to test the exampleSite and got an error at build.

```
/tmp> git clone https://github.com/LucasVadilho/heyo-hugo-theme heyo
cd heyo/exampleSite && hugo server --themesDir ../..
Cloning into 'heyo'...
remote: Enumerating objects: 351, done.
remote: Counting objects: 100% (351/351), done.
remote: Compressing objects: 100% (222/222), done.
remote: Total 351 (delta 143), reused 303 (delta 104), pack-reused 0
Receiving objects: 100% (351/351), 7.01 MiB | 3.25 MiB/s, done.
Resolving deltas: 100% (143/143), done.
Start building sites … 
hugo v0.92.2+extended linux/amd64 BuildDate=2023-01-31T11:11:57Z VendorInfo=ubuntu:0.92.2-1ubuntu0.1
Error: Error building site: failed to render pages: render of "page" failed: execute of template failed: template: _default/single.html:3:8: executing "_default/single.html" at <partial "head.html" .>: error calling partial: execute of template failed: template: _internal/opengraph.html:37:17: executing "_internal/opengraph.html" at <index $siteSeries ($name | urlize)>: error calling index: index of untyped nil
Built in 59 ms
exit 255
```

**Comment by LucasVadilho** [comment]
Hey @razvanf1, thanks for reporting!

The exampleSite works in the current hugo version (v0.120.4), if you want to update you hugo version you can get the instructions [here](https://github.com/gohugoio/hugo/releases).

However, I also fixed it for v0.92.2 (it was necessary to add the `series` taxonomy to `config.toml`), so you should be able to build it now! Could you try pulling the changes and building it again?

**Comment by razvanf1** [comment]
Thank you for your response!
I confirm that now it works on hugo v0.92.2.
